src/_stylesheets/_inform-inspire.scss

```
@@ -1,80 +1,154 @@
+$border-width: 2px;
+$arrow-width: 12px;
+
 .app-inform-inspire {
-  $padding-left: govuk-spacing(6);
-  $border-width: 2px;
-  $gap: $padding-left;
-  $arrow-width: 16px;
-  margin-top: govuk-spacing(1);
-  margin-bottom: govuk-spacing(4);
-  padding-left: $padding-left;
+  @include govuk-responsive-margin(6, $direction: 'bottom');
+  @include govuk-responsive-padding(6);
+  background-color: $app-light-blue;
+}
 
-  border-left: solid $border-width;
+// Reset margins in content for non-component elements eg: figures
+.app-inform-inspire__content > * {
+  margin-bottom: 0;
+}
 
-  &::before {
-    content: '';
-    box-sizing: border-box;
-    display: block;
-    width: $arrow-width;
-    height: $arrow-width;
-    margin-left: -$padding-left;
-    transform: rotate(45deg) translateX(-$border-width) translateY(-1px);
-    border-top: solid $border-width;
-    border-left: solid $border-width;
-    transform-origin: top left;
+.app-inform-inspire__content--flex {
+  display: flex;
+}
+
+// These modifiers could be looped over, however since there's only the need for
+// 2 cases: 2 items and 5 items, and we don't know what we'd want the gaps to be
+// for other cases, nor can we programmatically determine them right now, we
+// manually generate only these 2.
+.app-inform-inspire__content--flex-2-items {
+  gap: govuk-spacing(5);
+
+  & > * {
+    width: 50%;
   }
+}
 
-  &::after {
-    content: '';
-    box-sizing: border-box;
-    display: block;
-    width: $arrow-width;
-    height: $arrow-width;
-    margin-left: -$padding-left;
-    transform: rotate(-45deg) translateX(-$border-width) translateY(1px);
-    border-bottom: solid $border-width;
-    border-left: solid $border-width;
-    transform-origin: bottom left;
+.app-inform-inspire__content--flex-5-items {
+  gap: govuk-spacing(1);
+
+  & > * {
+    width: 20%;
   }
+}
+
+// Overriding .govuk-list's spacing so that it doesn't add extra space at the
+// bottom of the block and creates space between the content element above
+.app-inform-inspire__list {
+  @include govuk-responsive-margin(4, $direction: 'top');
+  margin-bottom: 0;
+}
+
+// Scale bar elements on the homepage
+.app-inform-inspire__bar {
+  @include govuk-font($size: 19, $weight: bold);
+  @include govuk-responsive-padding(4);
+  box-sizing: border-box;
+  width: 100%;
+  margin-bottom: govuk-spacing(1);
+  color: govuk-colour('white');
 
-  figure {
-    margin-inline: 0;
+  @media screen and (forced-colors: active), (-ms-high-contrast: active) {
+    outline: 1px solid Highlight;
   }
 
-  figure div {
-    width: 60%;
-    margin: 0 auto;
+  &:nth-of-type(1) {
+    width: calc(100% / 3);
+    background-color: govuk-colour('blue');
   }
 
-  figcaption {
-    margin-top: govuk-spacing(1);
+  &:nth-of-type(2) {
+    width: calc((100% / 3) * 2);
```

Could possibly be simplified.

```suggestion
    width: calc(100% * 0.666);
```